docs/documentation/getting-started.md

```diff
@@ -296,22 +91,24 @@ To use it, you will first have to create an account using the [signup page](http
   - If no server is supposed to be specified, just leave the field empty.
   - If a server is specified, a port has to be specified as well.
 
-**NOTE:** As mentioned in the section [missing features](#missing-features), Saros/I does currently not support the creation, management or deletion of XMPP accounts.
-As only people on your friends list can be invited to join your Saros session, you will have to create an account and add friends to your friends list through a different client.
-Any XMPP client (including Saros/E) can be used for this purpose.
-
-If you accidentally made a typo while entering your username or password, the created account entry can also only be changed or deleted through Saros/E (or by deleting the account store `~/.saros/config.dat`).
-Saros/E does not permit the deletion of the currently chosen account.
-If you only added one account and would like to remove it, you will have to add a second account (for example with random values) and choose this new account as the default.
-You can then delete the first account, add a new account with the right values, choose it as the correct default and delete the temporary account entry.
+- Click the "Connect" button (![connect icon](images/icons/connect.png)).
+- Choose the account you would like to connect to from the pop-up menu.
+{% endcapture %}
+{% include ide-tabs.html eclipse=eclipse intellij=intellij %}
 
-### Connecting to an XMPP Account
+### Step 3: Adding Contacts
 
-- Click the "Connect" button (![connect icon](images/icons-i/connect.png)).
-- Choose the account you would like to connect to from the pop-up menu.
+{% capture eclipse %}
+To add a contact to your list you need to **know his/her XMPP ID**.
 
-### Adding a Contact to an XMPP Account
+**After entering your partner's XMPP ID**, a message will be sent to
+your contact.
+Once s/he accepts your request you will be able to see his/her **online
```

Comment:
I would generally use singular they in such cases. But this is a matter of taste.
```suggestion
Once they accept your request, you will be able to see their **online
```